src/middleware/jira-symmetric-jwt-middleware.test.ts

```diff
@@ -3,13 +3,31 @@ import express, { Application, NextFunction, Request, Response } from "express";
 import { noop } from "lodash";
 import supertest from "supertest";
 import { getLogger } from "~/src/config/logger";
-import { jiraSymmetricJwtMiddleware } from "~/src/middleware/jira-symmetric-jwt-middleware";
+import { jiraSymmetricJwtMiddleware, setJiraAdminPrivileges } from "~/src/middleware/jira-symmetric-jwt-middleware";
 import { Installation } from "~/src/models/installation";
+import { getJiraClient } from "~/src/jira/client/jira-client";
 
+jest.mock("~/src/jira/client/jira-client", () => {
+	return {
+		getJiraClient: jest.fn()
+	};
+});
 
 jest.mock("config/feature-flags");
 const testSharedSecret = "test-secret";
 
+const getToken = ({
```

Comment:
This isnt a new function i just moved it up since i introduced a new describe block, felt better but meh